test/models/task_test.rb

```diff
@@ -59,17 +59,11 @@ def setup
     assert_equal ['foo', 'bar'], t.options
   end
 
-  test "should set initial status" do
+  test "should set initial and update status" do
     t = create_task status: nil
     assert_equal 'unresolved', t.reload.status
-  end
-
-  test "should add response to task" do
-    t = create_task
-    assert_equal 'unresolved', t.reload.status
-    at = create_annotation_type annotation_type: 'response'
-    create_field_instance annotation_type_object: at, name: 'response_test'
-    t.response = { annotation_type: 'response', set_fields: { response_test: 'test' }.to_json }.to_json
```

Comment:
I think we should keep this test... and at the end, assert that `t.responses.count > 0`, given that now in many parts of the code you replaced `t.status == 'resolved'` by `t.responses.count > 0`

Reply:
And before that, assert that `t.responses.count == 0`

Reply:
Ok

Reply:
Done